plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -119,9 +113,24 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
         return null
     }
 
+    /** Returns the output of the command executed in the container. */
+    fun execWithOutput(params: List<String>): String {
+        val args: List<String> = listOf(
```

Comment:
If the args are the same then don't duplicate it.

Reply:
Done.